custom/icds_reports/static/js/directives/month-filter/month-filter.directive.js

```diff
@@ -27,6 +28,12 @@ function MonthModalController($location, $uibModalInstance) {
     vm.selectedMonth = $location.search()['month'] !== void(0) ? parseInt($location.search()['month']) : new Date().getMonth() + 1;
     vm.selectedYear = $location.search()['year'] !== void(0) ? parseInt($location.search()['year']) : new Date().getFullYear();
 
+    if(isSDD && (vm.selectedYear<2019||vm.selectedYear==2019&&vm.selectedMonth==1)){
```

Comment:
add white space around operators

could you add brackets to the condition, will be more readable in future